doc/_templates/citing.html

```diff
@@ -5,8 +5,9 @@
 <h1>Citing Matplotlib</h1>
 <p>
 If Matplotlib contributes to a project that leads to a scientific publication,
-please acknowledge this fact by citing the project. You can use this
-BibTeX entry:
+please acknowledge this fact by citing the project. You can cite
+<a href ='http://ieeexplore.ieee.org/document/4160265/'>this paper</a> or
```

Comment:
... by using (as they're referring to the same object)

Reply:
Strongly suggest linking by DOI so doi.org gets to take care of any changes to ieee’s links...  http://doi.org/10.1109/MCSE.2007.55

Reply:
I think the English is fine as is, if anyone wants to change feel free to force push. Have changed to the doi.org link.